LeetCodeSolutions/UniquePaths/UniquePaths.cs

```diff
@@ -0,0 +1,47 @@
+
+
+namespace LeetCodeSolutions;
+public class UniquePathsSolution
+{
+    private int[][] AllocateSpaceForPascalTable(int m, int n)
+    {
+        int[][] table = new int[m][];
+        for (int i = 0; i < m; i++)
+        {
+            table[i] = new int[n];
+        }
+        return table;
+    }
+
+    private void PopulateOnesOfPascalTable(ref int[][] table)
+    {
+        for (int i = 0; i < table.Length; i++)
+        {
+            table[i][0] = 1;
+
+        }
+        for (int j = 0; j < table[0].Length; j++)
+        {
+            table[0][j] = 1;
+        }
+    }
+
+    private void PopulateTableWithSums(ref int[][] table)
+    {
+        for (int i = 1; i < table.Length; i++)
+        {
+            for (int j = 1; j < table[0].Length; j++)
+            {
+                table[i][j] = table[i - 1][j] + table[i][j - 1];
+            }
+        }
+    }
+
+    public int UniquePaths(int m, int n)
+    {
+        int[][] table = AllocateSpaceForPascalTable(m, n);
+        PopulateOnesOfPascalTable(ref table);
```

Comment:
Create an object and make member functions, don't need to keep passing refs around